### DISCUSSION TITLE
Copilot chat says it doesn't read my editor screen but it does

### DISCUSSION THREAD

**Discussion by adpposner** [community/community#69445] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I asked Copilot chat a question that was not directly related to the code in my editor and it gave a response using a line I had written five minutes earlier - since the line is pretty specific and has parameters set in it (by me), it was not scraped off the internet. When I asked Copilot chat about it, it says it can't read my editor, but clearly it can. I was asking it about Python Pandas usage and it dropped this line in an example: "--psm 11 -c language_model_penalty_non_dict_word=0.0...". In the attached screenshots, the one on the left has it mentioning that line, and in the one on the right it is stating that it does not see my editor screen. 

I should add that _nowhere_ has anything remotely like this appeared in my chat history. That said, I do have regular Copilot enabled in addition to chat.

So can it read my screen or not? This isn't particularly sensitive information but I'm a physician and I have used VS Code in the past to make quick notes after patient visits or on imaging studies, so this is kind of a big deal. Please let me know.
![2023-10-06 (1)](https://github.com/community/community/assets/34519115/06937053-4ac5-48f5-9cb8-dde9a66a4318)

**Comment by moayad-star** [comment]
In fact, I do not know what exactly is happening, but there are many possibilities, such as that he is actually reading what you have, or perhaps it is just a guess. Advice, go to the support center if you have any concerns, or you can search within the docs to determine exactly what is happening to you and its location, collect more information, and present it here or in a discussion  New
